boot/boot_serial/src/boot_serial.c

```diff
@@ -240,7 +240,7 @@ bs_list_img_ver(char *dst, int maxlen, struct image_version *ver)
  * List images.
  */
 static void
-bs_list(char *buf, int len)
+bs_list(void)
```

Comment:
There are conditionally compiled in calls, like bs_set, that still take both parameters.

Reply:
@de-nordic yes, but the `bs_set()` makes use of these parameters, the `bs_list()` doesn't. Am I not seeing something obvious here?

Reply:
Ah, sorry. I have looked at diffs quickly and missed that the bs_set is called from different function.